### ISSUE TITLE
Web Scrobbler stopped to work when Spotify Web Player is not the selected device

### ISSUE THREAD

**Issue by jckebs1** [web-scrobbler/web-scrobbler#2932]
**Describe the bug**

I always used the Web Scrobbler to edit tags. I never used the Spotify Web Player as audio source, but the Spotify Desktop Player. There is a feature called "Conect to a device" that allows you to select the audio source. If you play a song on Spotify Desktop and then open the Spotify Web Player, the Spotify Web Player will recognize that Spotify Desktop Player is the selected device, i.e., the audio source (see screenshot).

The Web Scrobbler used to recognize the track even the Spotify Web Player was not the audio source. And through this I was able to edit the tags.

This worked very well for a long, long time. But yesterday it stopped to work and I don't know why. Now the Web Scrobbler does not recognize the track in Spotify Web Player when the audio source is Spotify Desktop Player. Only when I put the Spotify Web Player as the audio source in the "Conect to a device" that Web Scrobbler works. This is a problem, because the Desktop software is better, has a better audio quality etc.

In short:

BEFORE: Spotify Desktop "master" -----> Spotify Web "slave" ---------> Web Scrobbler (edit option OK) 

NOW: Spotify Desktop "master" -----> Spotify Web "slave" ---------> Web Scrobbler (does not work) 

BUT: Spotify Web  ---------> Web Scrobbler (edit option OK) 

**Environment (please complete the following information)**
-   Windows
-   Browser: Opera
-   2.44.0

![image](https://user-images.githubusercontent.com/87746294/126436662-fbdd8fbe-0fea-4e3e-a492-0d06a06f3ed6.png)

**Comment by MisterZob** [comment]
Same here, with Chrome (Win10).
Also it seems the issue isn't limited to Spotify's desktop app, because it doesn't work with the iOS app either.

**Comment by jckebs1** [comment]
Now working! Awesome, man!
